backend/api/admin_views.py

```diff
@@ -30,81 +30,76 @@ def make_err_response(msg):
 
     return None
 
-@api_view(['GET'])
+@api_view(['GET', 'POST'])
 # special permissions needed because GET is usually readonly (and therefore visible to all),
 # but we don't want just anyone seeing all of our admins' info
 @permission_classes([IsAuthenticatedOrOptions])
```

Comment:
Looks good as long as authentication on GET still works. Also if everything is tested and still working :)